testnet/run_libp2p.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -f /usr/bin/etcdctl ]; then
+	./etc/nomos/etcd/install_etcd.sh
```

Comment:
Why are we installing things at startup rather than making a Docker image for this?

Reply:
I've tried to avoid adding `etcd` dependency to the node Dockerfile or wrapping `docker compose up` into another shell script. The ideal solution would be to build a node image, then use that image to build a new version that has all dependencies for the testnet. I didn't find a way to use an image A to build an image B without having A already built.

Reply:
The client package is like 17 MB, is that really worth adding all that complexity?

![image](https://github.com/logos-co/nomos-node/assets/2212681/f440c771-a4ad-4463-a837-2502b8dc0cfe)

https://packages.debian.org/bookworm/etcd-client

I'd rather keep things simple and just install it in one image.